src/python/TaskWorker/Actions/Splitter.py

```diff
@@ -40,6 +40,9 @@ def execute(self, *args, **kwargs):
                     totalUnits = int(totalUnits * sum(f['events'] for f in data.getFiles()) + 0.5)
                 splitparam['total_events'] = totalUnits
             elif kwargs['task']['tm_split_algo'] == 'Automatic':
+                # REST backwards compatibility fix
+                if 'seconds_per_job' in kwargs['task']['tm_split_args']:
+                    kwargs['task']['tm_split_args']['minutes_per_job'] = kwargs['task']['tm_split_args'].pop('seconds_per_job')
```

Comment:
maybe you want to divide the seconds by 60

Reply:
Also, remember t use // instead of / for the division, this file does not have "from future import division on top"

Reply:
This is more a stop-gap measure: the client is updated to provide a default in minutes, but REST still labels it as "seconds", so we shouldn't really divide. This is not meant to run old tasks with everything in seconds, but until REST is updated consistently.

TL:DR; I don't think we should divide here, then the client will have to pass units in seconds to the REST again.

Reply:
ah ok, I missed the client is already providing minutes!

Reply:
It does not explicitly, but I've changed the default setting to be in minutes already. There is no mention of units anywhere in the client…